runtime/stdlib/builtin.go

```diff
@@ -344,12 +215,67 @@ func BuiltinValues() StandardLibraryValues {
 			)
 		},
 		Kind: common.DeclarationKindEnum,
-	}
-
-	return StandardLibraryValues{
-		signatureAlgorithmValue,
-		hashAlgorithmValue,
-	}
+	},
+	{
+		Name: "BLS",
+		Type: blsContractType,
+		ValueFactory: func(inter *interpreter.Interpreter) interpreter.Value {
+			return interpreter.NewSimpleCompositeValue(
+				blsContractType.ID(),
+				blsContractStaticType,
+				blsContractDynamicType,
+				nil,
+				map[string]interpreter.Value{
+					blsAggregatePublicKeysFunctionName: interpreter.NewHostFunctionValue(
+						func(invocation interpreter.Invocation) interpreter.Value {
+							publicKeys := invocation.Arguments[0].(*interpreter.ArrayValue)
+
+							inter := invocation.Interpreter
+							getLocationRange := invocation.GetLocationRange
+
+							inter.ExpectType(
+								publicKeys,
+								sema.PublicKeyArrayType,
+								getLocationRange,
+							)
+
+							return invocation.Interpreter.BLSAggregatePublicKeysHandler(
+								inter,
+								getLocationRange,
+								publicKeys,
+							)
+						},
+						blsAggregatePublicKeysFunctionType,
+					),
+					blsAggregateSignaturesFunctionName: interpreter.NewHostFunctionValue(
+						func(invocation interpreter.Invocation) interpreter.Value {
+							signatures := invocation.Arguments[0].(*interpreter.ArrayValue)
```

Comment:
check this cast

Reply:
Added in c2379e793a00bc91eae86e1ab3bbd219cc89088d